composer.json

```diff
@@ -44,5 +44,10 @@
       "Cake\\Chronos\\Test\\": "tests"
     },
     "files": ["tests/TestCase.php"]
+  },
+  "scripts": {
+      "phpunit": "phpunit",
+      "phpcs": "phpcs",
```

Comment:
those should be the standard keys: "cs-check" and "cs-fix".
phpunit should be "test". See cakephp/app repo.